test/integration/verify-server-ca-cert.js

```diff
@@ -0,0 +1,50 @@
+'use strict'
+
+var getPort = require('get-port')
+
+getPort().then(function (port) {
+  var fs = require('fs')
+  var path = require('path')
+
+  var agent = require('../../').start({
+    serviceName: 'test',
+    serverUrl: 'https://localhost:' + port,
+    captureExceptions: false,
+    metricsInterval: 0,
+    centralConfig: false,
+    disableInstrumentations: ['https'], // avoid the agent instrumenting the mock APM Server
+    serverCaCert: path.join(__dirname, 'cert.pem') // self-signed certificate
+  })
+
+  var https = require('https')
+  var test = require('tape')
+
+  test('should allow self signed certificate', function (t) {
+    t.plan(2)
+
+    var cert = fs.readFileSync(path.join(__dirname, 'cert.pem'))
+    var key = fs.readFileSync(path.join(__dirname, 'key.pem'))
+
+    var server = https.createServer({ cert, key }, function (req, res) {
+      t.pass('server received client request')
+      res.end()
+    })
+
+    server.listen(port, function () {
+      agent.captureError(new Error('boom!'), function () {
```

Comment:
I would check if the callback is called with an error as this could be an indication that something with the server certificate was wrong:

```suggestion
      agent.captureError(new Error('boom!'), function (err) {
        t.error(err)
```

Then you could either ajust `t.plan` to `3`, or just drop the extra `t.pass` below.

Reply:
Done. I copied the test from `test/integration/allow-invalid-cert.js`, so I've made the change there also.